CommunityToolkit.HighPerformance/Streams/Sources/MemoryManagerOwner.cs

```diff
@@ -61,4 +61,14 @@ public Span<byte> Span
             return this.memoryManager.GetSpan().Slice(this.offset, this.length);
         }
     }
+
+    /// <inheritdoc/>
+    public Memory<byte> Memory
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        get
+        {
+            return this.memoryManager.Memory.Slice(this.offset, this.length);
+        }
```

Comment:
Use a one liner here as well.
```suggestion
        get => this.memoryManager.Memory.Slice(this.offset, this.length);
```